dora/src/bytecode/generator_tests.rs

```diff
@@ -1190,11 +1205,17 @@ fn gen_throw() {
                 Throw(r(0)),
                 RetVoid,
             ];
+            let expected_position = vec![(0, p(2, 38)), (6, p(2, 38)), (10, p(2, 23))];
             assert_eq!(expected, code);
+            assert_eq!(expected_position, fct.positions().to_vec());
         },
     );
 }
```

Comment:
I'm not sure, if it is wise to extend existing test cases, instead of _"duplicating"_ them. After some consideration I more and more lean to separate tests.

Reply:
Done.